pkg/deploy/util/util.go

```diff
@@ -72,6 +72,17 @@ func LabelForDeploymentConfig(config *deployapi.DeploymentConfig) string {
 	return fmt.Sprintf("%s/%s:%d", config.Namespace, config.Name, config.LatestVersion)
 }
 
+// ConfigSelector constructs a label selector for the provided DeploymentConfig
+func ConfigSelector(config *deployapi.DeploymentConfig) labels.Selector {
+	label := config.Template.ControllerTemplate.Selector
```

Comment:
@ironcladlou @smarterclayton  I think this is more sane to do rather than specifying our own standard label, what do you think?